near-sdk/src/environment/env.rs

```diff
@@ -183,6 +183,11 @@ pub fn block_timestamp() -> u64 {
     unsafe { sys::block_timestamp() }
 }
 
+/// Current block timestamp, i.e, number of non-leap-milliseconds since January 1, 1970 0:00:00 UTC.
+pub fn block_timestamp_ms() -> u64 {
+    block_timestamp() / 10u64.pow(6)
```

Comment:
```suggestion
    block_timestamp() / 1_000_000
```
pow isn't `const` afaik